src/controller/BlogController.java

```diff
@@ -61,6 +61,12 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 		
 		CRUDOperations crud=new CRUDOperations();
 		List<Blog> listblog = crud.createBlog(blog);
+		
+		//List<Blog> listblog1= crud.viewBlog(blog);
+		//List<Blog> listblog2= crud.updateBlog(blog);
+		
+		List<Blog> l1=crud.SortByDate();

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.